playbooks/roles/aws_logs_agent/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Download the AWS Logs Agent python script
+  get_url:
+    url: "{{ AWS_LOGS_AGENT_SCRIPT_URL }}"
+    dest: /tmp/aws-logs-agent.py
+    mode: "1777"
```

Comment:
I'm not sure what this script requires, but isn't this too permissive?

Reply:
Changed it to "0664"